lib/player.js

```diff
@@ -2856,7 +2856,7 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       return tracks;
     } else if (this.video_ && this.video_.src && this.video_.textTracks) {
       const textTracks = Array.from(this.video_.textTracks)
-          .filter((t) => t.mode != 'disabled');
+          .filter((t) => t.mode != 'disabled' && t.kind != 'metadata');
```

Comment:
I'm reverting the change to filter on `mode != 'disabled'`, so this will have to be updated soon.  Sorry for the inconvenience, and thanks for the bug reports!

Reply:
I've just merged the change I mentioned.  Please rebase at your earliest convenience.  The rest of the change looks good.  Thanks!

Reply:
I have done rebase. Thank you.